operate/services/manage.py

```diff
@@ -1519,24 +1525,38 @@ def fund_service_single_chain(  # pylint: disable=too-many-arguments,too-many-lo
                 contract_address=asset_address,
                 address=chain_data.multisig,
             )
+            if asset_address == ZERO_ADDRESS:
+                # also count the balance of the wrapped native asset
+                safe_balance += get_asset_balance(
+                    ledger_api=ledger_api,
+                    contract_address=WRAPPED_NATIVE_ASSET.get(chain, asset_address),
+                    address=chain_data.multisig,
+                )
```

Comment:
Why is this approach not taken into account also for the agent?

Reply:
Because in the case of Agent EOA, the agent doesn't swap xDAI for wxDAI and we don't want to it to happen that someone send wxDAI to the Agent EOA and the funding job no long funds xDAI in it, failing the agent to make transactions anymore.

Reply:
Seems quite custom logic, but ok.